chat-plugins/animals.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const http = require('http');
+
+exports.commands = {
+    animal: 'animals',
+    animals: function(target, room, user) {
+        let tarId = toId(target)
+        let validTargets = {
+            'cat': 'cat',
+            'otter': 'otter',
+            'dog': 'dog',
+            'bunny': 'bunny',
+            'pokemon': 'pokemon',
+            'kitten': 'kitten',
+            'puppy': 'puppy'
+            
+        };
+        let validTarget = validTargets[tarId];
```

Comment:
there should be a check if they do a target that's not one of the ``validTargets`` or something